src/main/java/ibas/inchelin/web/dto/notice/NoticeListResponse.java

```diff
@@ -1,13 +1,19 @@
 package ibas.inchelin.web.dto.notice;
 
-import java.util.List;
-
-public record NoticeListResponse(
-        List<NoticeResponse> notices
-) {
-    public record NoticeResponse(
-            Long noticeId,
-            String title,
-            String createdAt
-    ) {}
-}
+import ibas.inchelin.domain.notice.entity.Notice;
+import lombok.Getter;
+
+public class NoticeListResponse {
+
+    //공지 목록 조회를 위한 dto
+    @Getter
+    public static class NoticeListDto {
+        private Long noticeId;
+        private String title;
+
+        public NoticeListDto(Notice notice) {
+            this.noticeId = notice.getId();
+            this.title = notice.getTitle();
+        }
+    }
+}
```

Comment:
유지보수성을 위해 코드 스타일을 통일하고자 합니다.
따라서, 기존에 만들어둔 DTO는 수정 없이 그대로 사용해 주시면 좋겠습니다.
```suggestion
import java.util.List;

public record NoticeListResponse(
        List<NoticeResponse> notices
) {
    public record NoticeResponse(
            Long noticeId,
            String title,
            String createdAt
    ) {}
}
```